src/components/common/statbar/Stat.js

```diff
@@ -31,9 +31,11 @@ class Stat extends React.Component {
       );
     }
     return (
-      <DataCard className="stat">
-        {contentToShow}
-      </DataCard>
+      data !== '?' && (
+        <DataCard className="stat">
+          {contentToShow}
+        </DataCard>
+      )
     );
```

Comment:
Why not
```jsx
    if (!(data !== '?')) {
      return null;
    }
    return (
      <DataCard className="stat">
        {contentToShow}
      </DataCard>
    );
```

? And not sure why we're concerned with `data` instead of `contentToShow` either.

Reply:
Sure the suggestion you made works better, we can easily remove the condition if we ever need to.

Regarding the check on `data` from the tests i did when we render the `Stat` component it appears `contentToShow` is always assigned to a react component, thus it made more sense to check on the value of data (which will have `?` or some other value that will be displayed)